src/app/@ansyn/ansyn/components/ansyn-footer/ansyn-footer.component.less

```diff
@@ -63,7 +59,8 @@
 			.app-timeline-container {
 				ansyn-time-picker-container {
 					right: unset;
-					left: 55%;
+					left: 69%;
+					top: 8px
```

Comment:
Please move the "top" property to above definiion of `ansyn-time-picker-container` class (not  inside the rtl section)

Reply:
the &.rtl overrides the ansyn-time-picker-container class  above.
do you have an idea how to fix it?

Reply:
Why is it a problem? What is there to fix?
If you do the two changes that I suggested (undelete class +  move "top" property), is it not working?

Reply:
@sulimani55

Reply:
@sulimani55 OK I will look at it tomorrow

Reply:
Good morning. I pulled the branch and did the fixes, and it works. Here are the relevant classes

```
			ansyn-time-picker-container {
				top: 8px;
				right: 55%;
			}

	&.rtl {
		div.status-timeline-container {
			.app-timeline-container {
				ansyn-time-picker-container {
					right: unset;
					left: 69%;
				}
			}
		}
	}

```

Reply:
Another point, I don't why you are moving the time picker, but you moved it too far to the right. It hides the date of the timeline.

![image](https://user-images.githubusercontent.com/11027714/103624340-0da0a600-4f42-11eb-8f06-b0a28de19204.png)

Reply:
@sulimani55

Reply:
how do i make it go near the  date of the timeline? it changes to each screen.
on my screen it looks fine

Reply:
@ramtob

Reply:
@sulimani55 
An interesting question. 
I think you can do this by using `right` instead of `left`.
Something like:

right: calc(10% + 100px + 50px)

10% is (more of less) the width of the result table button
100px is (more of less) the width of the timeline date, on each side
50px is for margin

Or you can just leave the time picker where I placed it (55%) :)

Reply:
hey i tried your changes. can you pull my branch and take a look?
@ramtob